runtime/doc/repeat.txt

```diff
@@ -762,6 +765,35 @@ the command after changing the plugin help: >
 	:helptags path/start/foobar/doc
 	:helptags path/opt/fooextra/doc
 
+The messages that are in the lang/<lang_id>/LC_MESSAGES/foo.po file need to be
```

Comment:
do we need to specify in what <lang_id> should look like?

Reply:
In "mlang.txt" there is this nugget:
```Message files (vim.mo) have to be placed in "$VIMRUNTIME/lang/xx/LC_MESSAGES",
where "xx" is the abbreviation of the language (mostly two letters).
```

Perhaps I can link to *multilang-messages* here??